core/src/main/resources/application.yml

```diff
@@ -853,17 +853,24 @@ crn:
   region: us-west-1
 
 existingstackpatcher:
-  intervalhours: 24
+  intervalInHours: 24
   maxInitialStartDelayInHours: 2
```

Comment:
i think here, the default for `maxInitialStartDelayInHours` can be increased as well, as for now, there is a large change patches will start right after application starts

Reply:
Modified to 12